src/cli.ts

```diff
@@ -5,6 +5,10 @@ import program from "commander";
 import fs from "fs-extra";
 import { RecordReplayServer } from "./server";
 
+function commaSeparatedList(value: string, _dummyPrevious?: any) {
```

Comment:
I think you should be able to omit the second parameter:
```suggestion
function commaSeparatedList(value: string) {
```

Reply:
Interesting, I was following the example and it included the `dummyPrevious` argument. Will try it out and see. Thanks.